californium-core/src/main/java/org/eclipse/californium/core/network/stack/CoapUdpStack.java

```diff
@@ -99,6 +99,7 @@ public CoapUdpStack(final NetworkConfig config, final Outbox outbox) {
 				new ExchangeCleanupLayer(),
 				new ObserveLayer(config),
 				new BlockwiseLayer(config),
+				new ResponseTimeoutLayer(),
```

Comment:
If this is placed below the blockwise layer, I would guess, this will then apply to the "single block requests". Right?

Reply:
That's right.

We could imagine that we do not need this for "single block request" of transparent block transfer, because there is already a response timeout for the "parent" request.

But for block2 notification, there is no "parent" request.

Reply:
OK.